### ISSUE TITLE
Debug build of ZBS does not run.

### ISSUE THREAD

**Issue by jjvbsag** [pkulchenko/ZeroBraneStudio#966]
To get some working environment to have a look at #https://github.com/pkulchenko/wxlua/issues/20 I wanted to make a debug build. These are the scripts I use (after applying #958)

`build-zbs.sh`:
```sh
#!/bin/bash
cd /home/GitHub/pkulchenko/ZeroBraneStudio/build 
bash /home/GitHub/pkulchenko/ZeroBraneStudio/build/build-linux.sh all
```
`test-zbs.sh`:
```sh
#!/bin/bash
cd /home/GitHub/pkulchenko/ZeroBraneStudio/
/home/GitHub/pkulchenko/ZeroBraneStudio/zbstudio.sh
```
This works fine.

Then I change `build-zbs.sh` to
`…build-linux.sh all debug`

after a new build `test-zbs.sh` fails with
```
./src/aui/auibook.cpp(2286): assert "page_idx < m_tabs.GetPageCount()" failed in GetPage().
/home/GitHub/pkulchenko/ZeroBraneStudio/zbstudio.sh: line 20:  4391 Trace/breakpoint trap   (core dumped) bin/linux/$ARCH/lua src/main.lua zbstudio -cwd "$CWD" "$@"
```

Any suggestion, how to proceed?

---
everything using xubuntu 18.04.1 LTS

**Comment by pkulchenko** [comment]
> ./src/aui/auibook.cpp(2286): assert "page_idx < m_tabs.GetPageCount()" failed in GetPage().

Does it fail on launch or when you try to do something in the IDE?

**Comment by jjvbsag** [comment]
Direct on launch. I have also deactivated all my packages (e.g. zbssvn), still fails.

**Comment by pkulchenko** [comment]
Is this on the current master branch? Can you check if it runs on 1.80 or 1.70 and if so, can you bisect to the commit that makes it break?

I'm not sure why the current page would be beyond the number of pages, but I vaguely remember that there may be one place where it's done as the workaround. I couldn't find it, though, so it might have been removed already.

**Comment by pkulchenko** [comment]
@jjvbsag, should I close this? I can't reproduce this, but it could be resolved in the upgrade branch (see #999 ticket).

**Comment by jjvbsag** [comment]
@pkulchenko Just verified with branch `all-binaries-190c`using this script
```sh
#!/bin/bash
set -e
branch=all-binaries-190c
test -f ${branch}.zip || wget https://github.com/pkulchenko/ZeroBraneStudio/archive/${branch}.zip
rm -rf ZeroBraneStudio-${branch}
unzip ${branch}.zip
cd ZeroBraneStudio-${branch}/build
./build-linux.sh all debug
cd ..
./zbstudio.sh
```
No issue any more, you can close

**Comment by pkulchenko** [comment]
This has been implemented as part of the changes in #999 that have been merged.
